src/agent/debuglet.ts

```diff
@@ -580,6 +600,10 @@ export class Debuglet extends EventEmitter {
       }
     }
 
+    if (region) {
+      desc += region;
```

Comment:
There probably needs to be some additional formatting here.  eg: desc += ' region: ' + region;

Reply:
done